src/rgw/rgw_admin.cc

```diff
@@ -6196,11 +6197,13 @@ int main(int argc, const char **argv)
     int ret = br.cancel();
     if (ret < 0) {
       if (ret == -EBUSY) {
-	cerr << "There is ongoing resharding, please retry after " << g_conf()->rgw_reshard_bucket_lock_duration <<
-	     " seconds " << std::endl;
+	cerr << "There is ongoing resharding, please retry after " <<
+	  g_conf().get_val<uint64_t>("rgw_reshard_bucket_lock_duration") <<
```

Comment:
Can't we get the val from `store->ctx()` here as well, as we have done above?
```suggestion
	store->ctx()->_conf.get_val<uint64_t>("rgw_reshard_bucket_lock_duration") <<
```

Reply:
Fixed; thanks!